### PR TITLE
[shopsys] add variant availabilities to product detail

### PR DESCRIPTION
**Pull request by DavidOstrozlik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Variant availability is not shown on product detail
|New feature| Yes 
|BC breaks| No 
|Fixes issues| ... 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by grossmannmartin** [comment]
Hi, I've rebased this branch to current master and from my point of view can be merged. 
I don't consider this a BC Break, so if you agree, update the description, please.

I agree with @PetrHeinz, the icon would be better, but I prefer to merge as it is and find icon later (I tried to find suitable icons, but no luck so far)

File: docs/upgrade/UPGRADE-unreleased.md [hunk 1]
```diff
@@ -33,6 +33,17 @@ There you can find links to upgrade notes for other versions too.
 ### Application
 - follow instructions in [the separate article](upgrade-instructions-for-read-model-for-product-lists.md) to introduce read model for frontend product lists into your project ([#1018](https://github.com/shopsys/shopsys/pull/1018))
     - we recommend to read [Introduction to Read Model](/docs/model/introduction-to-read-model.md) article
+- if you want to show availability of variants add these lines to `.table-variants` in `Product/detail.html.twig` [#771](https://github.com/shopsys/shopsys/pull/771)
```
Comment (TomasLudvik) [inline docs/upgrade/UPGRADE-unreleased.md]:
```suggestion
- add these lines to `.table-variants` in `Product/detail.html.twig` to display availability on product detail page in administration [#771](https://github.com/shopsys/shopsys/pull/771)
```

File: docs/upgrade/UPGRADE-unreleased.md [hunk 2]
```diff
@@ -33,6 +33,17 @@ There you can find links to upgrade notes for other versions too.
...
+    ```
+        <th class="table-variants__cell">{{ 'Availability'|trans }}</th>
```
Comment (TomasLudvik) [inline docs/upgrade/UPGRADE-unreleased.md]: Change this to diff please

**Comment by grossmannmartin** [comment]
@DavidOstrozlik  is this still relevant?

**Comment by pk16011990** [comment]
ping @DavidOstrozlik

**Pull request merged**